src/phasorpy/cluster.py

```diff
@@ -0,0 +1,263 @@
+"""Cluster phasor coordinates.
+
+The `phasorpy.cluster` module provides functions to:
+
+- cluster phasor coordinates using Gaussian Mixture Model (GMM):
+    - :py:func:`phasor_cluster_gmm`
+
+"""
+
+from __future__ import annotations
+
+__all__ = ['phasor_cluster_gmm']
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._typing import Any, ArrayLike
+
+import numbers
+from collections.abc import Sequence
+
+import numpy
+from sklearn.mixture import GaussianMixture
+
+
+def phasor_cluster_gmm(
+    real: ArrayLike,
+    imag: ArrayLike,
+    scaling: float = 2 * numpy.sqrt(2),
+    /,
+    *,
+    clusters: int = 1,
+    skip_axis: int | Sequence[int] | None = None,
+    **kwargs: Any,
+) -> tuple[
+    tuple[float, ...],
+    tuple[float, ...],
+    tuple[float, ...],
+    tuple[float, ...],
+    tuple[float, ...],
+]:
+    """Return elliptic clusters in phasor coordinates using GMM.
+
+    Fit a Gaussian Mixture Model (GMM) to the provided phasor coordinates and
+    extract the parameters of ellipses that represent each cluster according
+    to [1]_.
+
+    Parameters
+    ----------
+    real : array_like
+        Real component of phasor coordinates.
+    imag : array_like
+        Imaginary component of phasor coordinates.
+    scaling: float, default = 2*sqrt(2)
+        Scaling factor for the radii of major and minor axes. By default, it is set
+        to 2√2, which corresponds to the scaling of eigenvalues for a 95% confidence
+        ellipse (2σ).
+    clusters : int, optional
+        Number of Gaussian distributions to fit to phasor coordinates.
+        Defaults to 1.
+    skip_axis : int or sequence of int, optional
+        Axes to skip in the data (useful for multi-dimensional arrays).
+    **kwargs
+        Additional keyword arguments passed to
+<<<<<<< HEAD
+        `sklearn.mixture.GaussianMixture`.
+        Common options include:
+
+        - covariance_type : {'full', 'tied', 'diag', 'spherical'}
+        - max_iter : int, maximum number of EM iterations
+        - random_state : int, for reproducible results
+=======
+        :py:class:`sklearn.mixture.GaussianMixture`.
+>>>>>>> 473d5da594f091661132a072c5cf886f666862a3
+
+    Returns
+    -------
+    center_real : tuple of float
+        Real component of ellipse centers.
+    center_imag : tuple of float
+        Imaginary component of ellipse centers.
+    radius_major : tuple of float
+        Major radii of ellipses.
+    radius_minor : tuple of float
+        Minor radii of ellipses.
+    angle : tuple of float
+        Rotation angles of major axes in radians, within range [0, π].
```

Comment:
So far, PhasorPy does not use Unicode characters in docstrings or code. Use `pi` instead of `π` or latex for math formulas.